index.html

```diff
@@ -12,33 +12,45 @@
     />
   </head>
   <body>
-    <div class="navbar">
+    <nav class="navbar">
       <ul>
         <li class="logo"><a href="#">OMSOO</a></li>
         <li class="nav-item">
           <a href="#"><i class="fas fa-bars"></i></a>
         </li>
       </ul>
-    </div>
+    </nav>
     <br />
-    <section class="header-container">
+    <header class="header-container">
       <div class="sub-container">
-        <h1 class="header-highlight-1">Hey There.I’m OMSOO</h1>
+        <h1 class="header-highlight-1">
+          Hey There. <br />
+          I’m OMSOO
+        </h1>
```

Comment:
- [ ] You misunderstand the previous reviewer requirement ,You have to use `header` tag then inside that there is `nav` tag. After that the next section should use `section` tag as you have used before. For example:-
`<body>`
   `<header>`
      `<nav>`
      ` ........`
      `</nav>`
   `</header>`
   `<section>`
        `....all intro....`
    `</section>`
`</body>`